features/storage/system_storage/SystemStorage.cpp

```diff
@@ -174,14 +167,20 @@ MBED_WEAK BlockDevice *BlockDevice::get_default_instance()
 
 #endif
 
-    return &default_bd;
-
 #else
-
+#define NO_BD 1
     return NULL;
 
 #endif
 
+#ifdef VERIFY_READ_DEF_BD
+    static VerifyReadBlockDevice verify_read_bd(&default_bd);
```

Comment:
this is the line with an error `default_bd` not defined. As this changes the code, how come this problem be on master?

@davidsaada Is default_bd defined for CY8CKIT target?